tpgtools/documentation.go

```diff
@@ -86,8 +86,12 @@ func mergeProperties(ga, beta []Property) []Property {
 }
 
 func generateResourceWebsiteFile(res *Resource, resources map[Version][]*Resource, version *Version) {
-
 	res = mergeResource(res, resources, version)
+
+	if len(res.DocSamples()) <= 0 {
```

Comment:
This is probably a good way to determine when we should use MMV1 or tpgtools for documentation right now, but could confuse someone who doesn't know about it. can we log that tpgtools is skipping doc generation and why here?

Reply:
done